Integrations/Devo_v2/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
+
+## [20.3.3] - 2020-03-18
```

Comment:
@jochman is it intended to add a version line if noting was added ? if not, please take a look because it happens a fuw times.